tests/ui/unittests.js

```diff
@@ -14,10 +12,68 @@ casper.test.begin('Basic test', {
             if (startedCount != passedCount) {
                 helpers.capture('unittest-failures.png');
             }
-        });
+        }, function () {
+            test.assert(false, 'Test timeout');
+        }, 10000);
 
         casper.run(function() {
             test.done();
         });
     },
+
+    tearDown: function testStatus() {
+        casper.echo('Getting test results...');
+
+        function logError(type, msg) {
+            console.log(casper.test.colorize(type, 'WARNING') + ' ' + msg);
```

Comment:
`casper.log` doesn't seem to output anything. Might be something to do with our logging settings.